All Task Completed :)

kalehege opened the pull request.

thinkverse commented:
Missing the `doctrine/dbal` package used for [renaming columns](https://laravel.com/docs/8.x/migrations#renaming-columns), hence the failing test. 🙂

kalehege commented:
> Missing the `doctrine/dbal` package used for [renaming columns](https://laravel.com/docs/8.x/migrations#renaming-columns), hence the failing test. 🙂

Even if I did this correctly before the first yran problem came up. Later, when the problem subsided, I removed my repo and fixed it again. I somehow forgot that package a second time. It took me 35min to figure out what the problem was anyway.
